modules/migration-creating-migration-plan-cam.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-creating-migration-plan-cam_{context}']
+= Creating a migration plan in the CAM web console
+
+You can create a migration plan in the CAM web console.
+
+.Prerequisites
+
+The CAM web console must contain the following:
+
+* Source cluster
+* Target cluster, which is added automatically during the CAM tool installation
+* Replication repository
+
+.Procedure
+
+. Log in to the CAM web console on the {product-title} 4 cluster.
+. In the *Plans* section, click *Add plan*.
+. Enter the *Plan name* and click *Next*.
++
+The *Plan name* can contain up to 253 lower-case alphanumeric characters (`a-z, 0-9`). It must not contain spaces or underscores (`_`).
+. Select a *Source cluster*.
+. Select a *Target cluster*.
+. Select a *Replication repository*.
+. Select the projects to be migrated and click *Next*.
```

Comment:
@jwmatthews 

Why does the UI say "Select projects to be migrated" when we call them "namespaces" elsewhere in the documentation?

Reply:
OpenShift term is 'projects'
Kubernetes calls them 'namespaces'

Terms are used interchangeably for most part, some times there are subtle reasons why it must be called a project, but for most part we use then interchangeably.

Note upstream tools like Velero will always refer to namespaces.

Reply:
OK, I will leave them as they are.